build.sbt

```diff
@@ -11,6 +11,10 @@ libraryDependencies ++= Seq(
 
   "com.lightbend.akka" %% "akka-stream-alpakka-slick" % "0.17",
 
+  "com.typesafe" % "config" % "1.3.2",
+  "com.sun.mail" % "javax.mail" % "1.6.1",
+
```

Comment:
useless empty line

Reply:
done (or will be done when resolving merge conflict either way)